packages/grafana-prometheus/src/querybuilder/parsing.test.ts

```diff
@@ -746,6 +746,43 @@ describe('buildVisualQueryFromString', () => {
       },
     });
   });
+
+  it('strips enclosing quotes', () => {
+    expect(buildVisualQueryFromString('counters_logins{app=\'frontend\', host=`localhost`}')).toEqual(
+      noErrors({
+        metric: 'counters_logins',
+        labels: [
+          {
+            op: '=',
+            value: 'frontend',
+            label: 'app',
+          },
+          {
+            op: '=',
+            value: 'localhost',
+            label: 'host'
+          }
+        ],
+        operations: [],
+      })
+    );
+  });
+
+  it('leaves escaped quotes inside string', () => {
+    expect(buildVisualQueryFromString('counters_logins{app="fron\"\"tend"}')).toEqual(
```

Comment:
In this test there is an error. I think the query should be provided like below
`counters_logins{app="fron\"\"tend"}` and of course, the expectation should be the same. 
Could you please make that change?

Reply:
Sorry bit confused with this comment, is that not what I have? I can't see any differences